<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI</class>
 <widget class="QWidget" name="GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="pager">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="DialPage" name="page1"/>
     <widget class="MenuPage" name="page2"/>
     <widget class="LastPage" name="page3"/>
     <widget class="DBConnectPage" name="page4"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DialPage</class>
   <extends>QWidget</extends>
   <header location="global">DialPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MenuPage</class>
   <extends>QWidget</extends>
   <header location="global">MenuPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LastPage</class>
   <extends>QWidget</extends>
   <header location="global">LastPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DBConnectPage</class>
   <extends>QWidget</extends>
   <header location="global">DBConnectPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
